packages/api/src/beacon/routes/events.ts

```diff
@@ -5,7 +5,7 @@ import {RouteDef, TypeJson} from "../../utils/index.js";
 
 // See /packages/api/src/routes/index.ts for reasoning and instructions to add new routes
 
-export enum EventType {
+export const enum EventType {
```

Comment:
@dapplion What was the reason for changing this to a `const` enum besides just the minor reduction in generated code? The problem with const enums are that they are completely removed during compilation which means others using the library can not reuse them, see [discord](https://discord.com/channels/593655374469660673/667606053877317642/1065521781026332682)

Reply:
IMO this should be reinstated as an enum

Reply:
Also re-exported as necessary to allow for downstream reuse

Reply:
changed in #5042, seems to be re-exported already so we should be fine there

Reply:
Potentially unwanted side-effect from attempting to make EventType from API + ChainEvent work for the same handlers. Happy to revert it to enum